src/SkyImages.jl

```diff
@@ -19,7 +19,11 @@ export break_blob_into_tiles, break_image_into_tiles
 export get_local_sources
 export stitch_object_tiles
 
```

Comment:
oops, yeah I meant to remove that. I'll do it on master.

It won't immediately change anything since Grid is still pulled in by SloanDigitalSkySurvey.